webpack.config.js

```diff
@@ -4,7 +4,10 @@ const webpack = require('webpack');
 const path = require('path');
 
 module.exports = {
-    entry: './static/src/javascripts/boot-webpack.js',
+    entry: [
+        './static/src/javascripts/boot.js',
+        './static/src/javascripts/bootstraps/standard/main.js',
```

Comment:
yeah i think webpack will try to bundle what it sees as the most efficient, this will override that. i wonder if it _is_ more efficient?